File: php-sdk/src/triplink/vcc/card/client/query/AuthInfo.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/4
 * Time: 14:33
 */
namespace triplink\vcc\card\client\query;
class AuthInfo
{
    /**
     * 发生时间
     */
    var $occurTime;

    /**
     * 交易唯一号
     */
    var $transId;

    /**
     * 交易币种
     */
    var $transCurrency;

    /**
     * 交易币种金额
     */
    var $transCurrencyAmt;

    /**
     * 卡币种
     */
    var $localCurrency;

    /**
     * 卡本币种交易金额
     */
    var $localCurrencyAmt;

    /**
     * 交易响应码
     */
    var $respCode;

    /**
     * 交易响应码描述
     */
    var $respCodeDesc;

    /**
     * 授权码
     */
    var $approveCode;

    /**
     * 交易拒绝原因
     */
    var $declineReason;

    /**
     * 卡交易状态
     */
    var $status;

    /**
     * 交易地国家代码
     */
    var $countryCode;

    /**
     * 信息类型
     */
    var $messageType;

    /**
     * 信息类型描述
     */
    var $messageTypeDesc;

    /**
     * 扣款商户信息
     * 参考TransMerchantInfo
     */
    var $transMerchantInfo;

    /**
     * 用于今后的扩展
     */
    var $transExtra;

    /**
     * @return mixed
     */
    public function getOccurTime()
    {
        return $this->occurTime;
    }

    /**
     * @param mixed $occurTime
     */
    public function setOccurTime($occurTime): void
    {
        $this->occurTime = $occurTime;
    }

    /**
     * @return mixed
     */
    public function getTransId()
    {
        return $this->transId;
    }

    /**
     * @param mixed $transId
     */
    public function setTransId($transId): void
    {
        $this->transId = $transId;
    }

    /**
     * @return mixed
     */
    public function getTransCurrency()
    {
        return $this->transCurrency;
    }

    /**
     * @param mixed $transCurrency
     */
    public function setTransCurrency($transCurrency): void
    {
        $this->transCurrency = $transCurrency;
    }

    /**
     * @return mixed
     */
    public function getTransCurrencyAmt()
    {
        return $this->transCurrencyAmt;
    }

    /**
     * @param mixed $transCurrencyAmt
     */
    public function setTransCurrencyAmt($transCurrencyAmt): void
    {
        $this->transCurrencyAmt = $transCurrencyAmt;
    }

    /**
     * @return mixed
     */
    public function getLocalCurrency()
    {
        return $this->localCurrency;
    }

    /**
     * @param mixed $localCurrency
     */
    public function setLocalCurrency($localCurrency): void
    {
        $this->localCurrency = $localCurrency;
    }

    /**
     * @return mixed
     */
    public function getLocalCurrencyAmt()
    {
        return $this->localCurrencyAmt;
    }

    /**
     * @param mixed $localCurrencyAmt
     */
    public function setLocalCurrencyAmt($localCurrencyAmt): void
    {
        $this->localCurrencyAmt = $localCurrencyAmt;
    }

    /**
     * @return mixed
     */
    public function getRespCode()
    {
        return $this->respCode;
    }

    /**
     * @param mixed $respCode
     */
    public function setRespCode($respCode): void
    {
        $this->respCode = $respCode;
    }

    /**
     * @return mixed
     */
    public function getRespCodeDesc()
    {
        return $this->respCodeDesc;
    }

    /**
     * @param mixed $respCodeDesc
     */
    public function setRespCodeDesc($respCodeDesc): void
    {
        $this->respCodeDesc = $respCodeDesc;
    }

    /**
     * @return mixed
     */
    public function getApproveCode()
    {
        return $this->approveCode;
    }

    /**
     * @param mixed $approveCode
     */
    public function setApproveCode($approveCode): void
    {
        $this->approveCode = $approveCode;
    }

    /**
     * @return mixed
     */
    public function getDeclineReason()
    {
        return $this->declineReason;
    }

    /**
     * @param mixed $declineReason
     */
    public function setDeclineReason($declineReason): void
    {
        $this->declineReason = $declineReason;
    }

    /**
     * @return mixed
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * @param mixed $status
     */
    public function setStatus($status): void
    {
        $this->status = $status;
    }

    /**
     * @return mixed
     */
    public function getCountryCode()
    {
        return $this->countryCode;
    }

    /**
     * @param mixed $countryCode
     */
    public function setCountryCode($countryCode): void
    {
        $this->countryCode = $countryCode;
    }

    /**
     * @return mixed
     */
    public function getMessageType()
    {
        return $this->messageType;
    }

    /**
     * @param mixed $messageType
     */
    public function setMessageType($messageType): void
    {
        $this->messageType = $messageType;
    }

    /**
     * @return mixed
     */
    public function getMessageTypeDesc()
    {
        return $this->messageTypeDesc;
    }

    /**
     * @param mixed $messageTypeDesc
     */
    public function setMessageTypeDesc($messageTypeDesc): void
    {
        $this->messageTypeDesc = $messageTypeDesc;
    }

    /**
     * @return mixed
     */
    public function getTransMerchantInfo()
    {
        return $this->transMerchantInfo;
    }

    /**
     * @param mixed $transMerchantInfo
     */
    public function setTransMerchantInfo($transMerchantInfo): void
    {
        $this->transMerchantInfo = $transMerchantInfo;
    }

    /**
     * @return mixed
     */
    public function getTransExtra()
    {
        return $this->transExtra;
    }

    /**
     * @param mixed $transExtra
     */
    public function setTransExtra($transExtra): void
    {
        $this->transExtra = $transExtra;
    }


}

?>

File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/V1/AuthInfo.java
package com.ctrip.ccard.creditcard.vcc.bean.V1;

/**
 * Description:
 */
public class AuthInfo {
    /**
     * 发生时间
     */
    private String occurTime;

    /**
     * 交易唯一号
     */
    private String transId;

    /**
     * 交易币种
     */
    private String transCurrency;

    /**
     * 交易币种金额
     */
    private String transCurrencyAmt;

    /**
     * 卡币种
     */
    private String localCurrency;

    /**
     * 卡本币种交易金额
     */
    private String localCurrencyAmt;

    /**
     * 交易响应码
     */
    private String respCode;

    /**
     * 交易响应码描述
     */
    private String respCodeDesc;

    /**
     * 授权码
     */
    private String approveCode;

    /**
     * 交易拒绝原因
     */
    private String declineReason;

    /**
     * 卡交易状态
     */
    private String status;

    /**
     * 交易地国家代码
     */
    private String countryCode;

    /**
     * 信息类型
     */
    private String messageType;

    /**
     * 信息类型描述
     */
    private String messageTypeDesc;

    /**
     * 扣款商户信息
     */
    private TransMerchantInfo transMerchantInfo;

    /**
     * 用于今后的扩展
     */
    private String transExtra;

    public String getOccurTime() {
        return occurTime;
    }

    public void setOccurTime(String occurTime) {
        this.occurTime = occurTime;
    }

    public String getTransId() {
        return transId;
    }

    public void setTransId(String transId) {
        this.transId = transId;
    }

    public String getTransCurrency() {
        return transCurrency;
    }

    public void setTransCurrency(String transCurrency) {
        this.transCurrency = transCurrency;
    }

    public String getTransCurrencyAmt() {
        return transCurrencyAmt;
    }

    public void setTransCurrencyAmt(String transCurrencyAmt) {
        this.transCurrencyAmt = transCurrencyAmt;
    }

    public String getLocalCurrency() {
        return localCurrency;
    }

    public void setLocalCurrency(String localCurrency) {
        this.localCurrency = localCurrency;
    }

    public String getLocalCurrencyAmt() {
        return localCurrencyAmt;
    }

    public void setLocalCurrencyAmt(String localCurrencyAmt) {
        this.localCurrencyAmt = localCurrencyAmt;
    }

    public String getRespCode() {
        return respCode;
    }

    public void setRespCode(String respCode) {
        this.respCode = respCode;
    }

    public String getRespCodeDesc() {
        return respCodeDesc;
    }

    public void setRespCodeDesc(String respCodeDesc) {
        this.respCodeDesc = respCodeDesc;
    }

    public String getApproveCode() {
        return approveCode;
    }

    public void setApproveCode(String approveCode) {
        this.approveCode = approveCode;
    }

    public String getDeclineReason() {
        return declineReason;
    }

    public void setDeclineReason(String declineReason) {
        this.declineReason = declineReason;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public void setCountryCode(String countryCode) {
        this.countryCode = countryCode;
    }

    public String getMessageType() {
        return messageType;
    }

    public void setMessageType(String messageType) {
        this.messageType = messageType;
    }

    public String getMessageTypeDesc() {
        return messageTypeDesc;
    }

    public void setMessageTypeDesc(String messageTypeDesc) {
        this.messageTypeDesc = messageTypeDesc;
    }

    public TransMerchantInfo getTransMerchantInfo() {
        return transMerchantInfo;
    }

    public void setTransMerchantInfo(TransMerchantInfo transMerchantInfo) {
        this.transMerchantInfo = transMerchantInfo;
    }

    public String getTransExtra() {
        return transExtra;
    }

    public void setTransExtra(String transExtra) {
        this.transExtra = transExtra;
    }
}
